src/seedu/addressbook/AddressBook.java

```diff
@@ -556,7 +556,7 @@ private static int extractTargetIndexFromDeletePersonArgs(String rawArgs) {
      * @param index to check
      * @return whether it is valid
      */
-    private static boolean isDisplayIndexValidForLastPersonListingView(int index) {
+    private static boolean isIndexValid(int index) {
```

Comment:
you have made the method name quite generic, if the behavior is same across the code base, may as well update the comments to reflect it. If it is not the same across, may be, it helps to retain the specific name as used in the given codebase.